Dockerfile

```diff
@@ -20,6 +20,10 @@ COPY . .
 
 RUN pip install -e . --no-cache-dir
 
+RUN groupadd -g 1000 nonroot && \
```

Comment:
Some container services (such as OpenShift) run with random users, but assume that everything can be accessed by the `root` group which in contrast to the `root` user does not have any special permissions (see https://docs.openshift.com/enterprise/3.2/creating_images/guidelines.html#openshift-enterprise-specific-guidelines -> Support Arbitrary User IDs). Hence, we should add the user to the root group and not create a new group.

Reply:
@ortsaCniveK have you seen this comment?

Reply:
Sorry, was caught up with the holidays and work.  The root reason was to try and clean up the permissions of the`__pycache__` folder that's created automatically as it has caused issues in mounting volumes as in [this issue](https://github.com/RasaHQ/rasa/issues/4653) , but I guess you can either suppress its creation [here by changing the line to `python -B -m rasa_sdk`](https://github.com/RasaHQ/rasa-sdk/blob/17eb70187555c8656336f9f21dcb2eb18655f455/entrypoint.sh#L14) (not necessarily ideal for some users due to performance implications) or just remember to `sudo rm -rf __pycache__` if someone runs into issues, which I don't think is intuitive either.

Reply:
you mean running `sudo rm -rf __pycache__` before they mount the volume?

Reply:
correct; I think that it could potentially be exclusively a Linux issue, as I've also had similar issues running interactive mode with failure to create `story_graph.dot` and the resulting interactive story files.

Perhaps we simply need to is just use the non-root `$UID` for the current user?  I'll investigate to see what we could change to do that.

Reply:
> Perhaps we simply need to is just use the non-root $UID for the current user?

Yep, let's try that. That's what we are currently also doing in the rasa image. Can you share how the permissions of your mounted directory look like?